recipes/jupyter-starters-reinforcement-learning-dennybritz/build_starter.py

```diff
@@ -0,0 +1,93 @@
+"""build a jupyter-starter
```

Comment:
Can you clarify on the license of these scripts? Have you written them for the recipe or do they come from upstream?

Reply:
nah, i wrote'm here. If this ends up decent, I might try to get them into something grayskull adjacent. Will update.

Reply:
added conda-forge bsd to both